### DISCUSSION TITLE
Custom tagging model generates blank pos_ output

### DISCUSSION THREAD

**Discussion by funghetto** [explosion/spaCy#11598] [Help: Coding & Implementations] [answered]
I've trained my model (for more info see [my old issue](https://github.com/explosion/spaCy/discussions/11534).
On a little testing program:

```
import spacy

nlp = spacy.load("it_core_news_sm")
nlp2 = spacy.load(r"C:\Users\Michele\OneDrive\Documents\C\Programmaraffa2\Trainingmodel\UD_Italian-MarkIT-r2.10\Fine\model-last")

doc = nlp("Roma è la capitale d'Italia.")
doc2 = nlp2("Roma è la capitale d'Italia.")

for l in doc:
    print(l.pos_)
for k in doc2:
    print(k.pos_)
```

I'm getting this output:

```
PROPN
AUX
DET
NOUN
ADP
PROPN
PUNCT







```
Can someone give me any clue on why it is not tagging the phrase?

**Comment by adrianeboyd** [comment, marked as answer]
Check that your pipeline contains a `morphologizer`. Using the default spacy CoNLL-U converter, XPOS is saved as `token.tag_` and UPOS is saved as `token.pos_`. `token.tag_` is output by the `tagger` component and `token.pos_` + `token.morph` are output by the `morphologizer` component.
